content/en/legacy/flux/references/daemon.md

```diff
@@ -0,0 +1,113 @@
+---
+type: legacy
+title: Daemon (fluxd)
+---
+
+## Summary
+
+Flux daemon (`fluxd`, aka Flux agent) allows automation of application deployments and version control of cluster configuration.
+Version controlling of cluster manifests provides reproducibility and a historical trail of events.
+
+### Responsibilities
+
+#### Continuous Deployment
+
+1. Flux daemon monitors user git repo Kubernetes manifests for
+   changes, which it then deploys to the cluster.
+
+2. Flux daemon monitors container registry for running container image
+   updates. Detection of an image change (running container image tag
+   vs container registry image tag) triggers k8s manifest update, which
+   is committed to the user git repository, then deployed to the
+   Kubernetes cluster.
+
+#### Deployment approaches
+        
+1. Automate vs Deautomate
+
+    Deployment happens automatically when a new image tag is
+    detected. Deautomated deployment will not proceed until
+    manually released (through the CLI tool `fluxctl`).
+
+2. Lock vs Unlock
+
+    Deployment is pinned to a particular image tag.
+    New deployment will not proceed upon triggered release.
+
+## Setup and configuration
+
+`fluxd` requires setup and offers customization though a multitude of flags.
+
```

Comment:
This table is not particularly readable:

<img width="1345" alt="Screen Shot 2021-06-02 at 3 27 32 PM" src="https://user-images.githubusercontent.com/3286998/120540864-2741fe80-c3b7-11eb-9706-b39e6cef7ab6.png">

The old look was like this:
<img width="786" alt="Screen Shot 2021-06-02 at 3 28 35 PM" src="https://user-images.githubusercontent.com/3286998/120540947-404aaf80-c3b7-11eb-9e28-f4ce865d7b27.png">